test/env/certmanager.go

```diff
@@ -28,7 +28,7 @@ import (
 // patchCMConversionCRDs patches the passed CRD objects to include the URL and
 // CA of the cert-manager webhook serving certificate. Only patches
 // cert-manager CRDs.
-func patchCMConversionCRDs(crds []apiextensionsv1.CustomResourceDefinition, url string, caPEM []byte) {
+func patchCMConversionCRDs(crds []*apiextensionsv1.CustomResourceDefinition, url string, caPEM []byte) {
```

Comment:
Since controller-runtime 0.11, `WaitForCRDs` expects a slice of CRD pointers.
 * https://github.com/kubernetes-sigs/controller-runtime/pull/1667